### PR TITLE
UICIRC-797: Add RTL/Jest tests for `src/settings/Models/FixedDueDateSchedule` folder

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add RTL/Jest testing for `src/settings/Models/FixedDueDateSchedule` folder.

## Refs
https://issues.folio.org/browse/UICIRC-797

## Screenshots
image

### CODE REVIEW

**Review by artem-blazhko — COMMENTED** [review]
Is it possible to test cases when `condA` and `condB` equal false?

https://github.com/folio-org/ui-circulation/blob/fe74e38dae85a7925618d333fb6069bb503af85a/src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.js#L35

File: src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.test.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    };
+
+    describe('when all props are valid', () => {
+      const scheduleInstance = new FixedDueDateSchedule(validScheduleProps);
+
+      it('should have correct values', () => {
+        expect(scheduleInstance.id).toBe(validScheduleProps.id);
+        expect(scheduleInstance.name).toBe(validScheduleProps.name);
+        expect(scheduleInstance.metadata).toEqual(expect.objectContaining(validScheduleProps.metadata));
+        expect(scheduleInstance.schedules).toEqual(expect.objectContaining(validScheduleProps.schedules));
```
[the remark this answers is not in the document]
Comment (Dmitriy-Litvinenko) [reply src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.test.js:69]: And similar problem above

**Comment by Stanislau-Snapok** [comment]
> Is it possible to test cases when `condA` and `condB` equal false?
> 
> https://github.com/folio-org/ui-circulation/blob/fe74e38dae85a7925618d333fb6069bb503af85a/src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.js#L35

I believe we can simulate this case here, but it looks it's possible only with invalid schedules.

File: src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/Models/FixedDueDateSchedule/FixedDueDateSchedule.test.js]: good point, thanks!

File: src/settings/Models/FixedDueDateSchedule/Schedule.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+describe('Schedule', () => {
+  const testData = {
+    from: 'testFrom',
+    to: 'testTo',
+    due: 'testDue',
+  };
+
+  commonClassCheckForEachProp(Schedule, testData);
+
+  commonClassCheckForAllProps(Schedule, testData);
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/Models/FixedDueDateSchedule/Schedule.test.js:19]: As I remember we make a decision to test Classes by each prop separately and for all at once.
I added new util `commonClassCheckWithoutProps` to check class without any data and with ability to skip any props (for example props that present instance of other Class). Could you please check it?

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**